sql/moz-fx-data-shared-prod/external_derived/chrome_extensions_v1/schema.yaml

```diff
@@ -68,3 +68,8 @@ fields:
   type: BOOLEAN
   description: |-
     Featured - Indicates if the featured badge is present, which indicates the extension follows recommended practices for Chrome extensions.
+- name: verified_domain
+  mode: NULLABLE
+  type: STRING
```

Comment:
Shouldn't this be Boolean?

Reply:
Apologies, wrong field. Should `has_verified_domain` be added to the schema?

Reply:
ok, I see. `has_verified_domain` only gets added in the view. Makes sense it's not here.